### ISSUE TITLE
fetchOrder not working - Binanceus

### ISSUE THREAD

**Issue by crossx4real** [ccxt/ccxt#6492]
- OS: Windows 10
- Programming Language version: Node Js
- CCXT version:
- Exchange: binanceus
- Method: fetchOrder

Hi i am receiving this error when ever i try to check the status of an order by calling exchaneg.fetchOrder(id). I've use this same line on coinbasepro and it works not sure why binance is asking for a symbol.

i've also tried querying using this format  "fetchOrder (id, symbol = undefined, {})" with the same issue

"UnhandledPromiseRejectionWarning: ArgumentsRequired: binanceus fetchOrder requires a symbol argument at binanceus.fetchOrder"

**Comment by kroitor** [comment]
> ArgumentsRequired: binanceus fetchOrder requires a symbol argument at binanceus.fetchOrder

This exception literally says that you have to supply a defined symbol argument to get the order:

```JavaScript
const orderId = '123456' // your order id here
const symbol = 'ETH/BTC' // the symbol that corresponds to that order
const order = await exchange.fetchOrder (orderId, symbol)
```

↑ This is documented in the Manual.

Hope that answers your question.

**Comment by crossx4real** [comment]
Igor, as always thanks for the quick response, i think i have solved the issue and thanks for super fast response.

    On Sunday, February 16, 2020, 05:29:30 AM EST, Igor Kroitor <notifications@github.com> wrote:  
 
 
Closed #6492.
